task/05-regex-tasks.js

```diff
@@ -31,7 +31,7 @@
  * @return {RegExp}
  */
 function getRegexForGuid() {
-   throw new Error('Not implemented');
+   return new RegExp('{[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}}');
```

Comment:
can be a little bit easier with flags
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/RegExp